content/en/blog/2024/v1-apis/index.md

```diff
@@ -0,0 +1,89 @@
+    * At this time, Istio lacks usage data to support the need for this feature.
+* `accessLogging.filter`
+    * If specified, this filter will be used to select specific requests/connections for logging.
+    * This feature is based on a relatively new feature in Envoy, and Istio needs to further develop the use case and implementation before graduating it to `v1`.
+* `tracing.useRequestIdForTraceSampling`
+    * This value is true by default. The format of this Request ID is specific to Envoy, and if the Request ID generated by the proxy that receives user traffic first is not specific to Envoy, Envoy will break the trace because it cannot interpret the Request ID. By setting this value to false, we can prevent [Envoy from sampling based on the Request ID](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/observability/tracing#trace-context-propagation).
+    * There is not a strong use case for making this configurable through the Telemetry API.
+
+Please share your feedback on any of the above fields by creating [issues](https://github.com/istio/istio/issues).
+
+## Overview of Istio CRDs
+
+| API | Versions |
+|---------|------------------------|
+| **Telemetry APIs** |
+| [Telemetry](/docs/reference/config/telemetry/) |  `v1`, `v1alpha1` |
+| **Security APIs** |
+| [Authorization Policy](/docs/reference/config/security/authorization-policy/) |  `v1`, `v1beta1` |
+| [Peer Authentication](/docs/reference/config/security/peer_authentication/) |  `v1`, `v1beta1` |
+| [Request Authentication](/docs/reference/config/security/request_authentication/) |  `v1`, `v1beta1` |
+| **Networking APIs** |
+| [Destination Rule](/docs/reference/config/networking/destination-rule/) |  `v1`, `v1beta1`, `v1alpha3` |
+| [Envoy Filter](/docs/reference/config/networking/envoy-filter/) |  `v1alpha3` |
+| Istio [Gateway](/docs/reference/config/networking/gateway/) |  `v1`, `v1beta1`, `v1alpha3` |
+| [Proxy Config](/docs/reference/config/networking/proxy-config/) |  `v1beta1` |
+| [Service Entry](/docs/reference/config/networking/service-entry/) |  `v1`, `v1beta1`, `v1alpha3` |
+| [Sidecar](/docs/reference/config/networking/sidecar/) |  `v1`, `v1beta1`, `v1alpha3` |
+| [Virtual Service](/docs/reference/config/networking/virtual-service/) |  `v1`, `v1beta1`, `v1alpha3` |
+| [Workload Entry](/docs/reference/config/networking/workload-entry/) |  `v1`, `v1beta1`, `v1alpha3` |
+| [Workload Group](/docs/reference/config/networking/workload-group/) |  `v1`, `v1beta1`, `v1alpha3` |
+| **Extension APIs** |
+| [Wasm Plugin](/docs/reference/config/proxy_extensions/wasm-plugin/) |  `v1alpha1` |
+
+## Using the `v1` Istio APIs
+
+As seen above, there are a few non-`v1` APIs and fields. The Envoy Filter, Proxy Config and Wasm Plugin aren’t `v1`, and within the Telemetry API, there are 3 non-`v1` fields.
+
+Since Istio maintains the same schema across API versions, users can utilize non-`v1` fields in their `v1` Custom Resources.
```

Comment:
I don't think this applies to Envoy Filter, Proxy Config and Wasm Plugin, so felt odd to have this after it.

Reply:
I have updated the wording, does it make more sense now?

Reply:
I think it may make sense to cover non-`v1` fields in their `v1` Custom Resources first, then cover non-v1 CRs next which is `EnvoyFilter` etc.

Reply:
PTAL, I have made changes to give more insights.

Reply:
LGTM; I understand now that it's not a decision on our part, it's just a side effect of how Kubernetes works.